cocotb/bus.py

```diff
@@ -36,10 +36,7 @@ def _build_sig_attr_dict(signals):
     if isinstance(signals, dict):
         return signals
     else:
-        sig_to_attr = {}
-        for sig in signals:
-            sig_to_attr[sig] = sig
-        return sig_to_attr
+        return dict([(sig, sig) for sig in signals])
```

Comment:
Why not go one further?
```suggestion
        return {sig: sig for sig in signals}
```

Reply:
Always happy to learn new tricks.
Changed as suggested.